sos/report/plugins/ceph_mon.py

```diff
@@ -51,6 +51,11 @@ def setup(self):
             "ceph osd erasure-code-profile ls",
             "ceph osd crush show-tunables",
             "ceph osd crush dump"
+            "ceph mgr dump",
```

Comment:
Maybe add a note why these are here instead of ceph mgr.

Reply:
ack